criu/tty.c

```diff
@@ -2342,13 +2342,14 @@ int tty_prep_fds(void)
 
 	if (!isatty(STDIN_FILENO))
 		pr_info("Standard stream is not a terminal, may fail later\n");
-	else
+	else {
```

Comment:
Could you please add braces to both branches of a conditional statement?

https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces